games/armagetronad/Portfile

```diff
@@ -0,0 +1,291 @@
+description         $pretty_name, a 3-D Tron-like light cycle game
+long_description    $pretty_name is a 3-D snake game, based on the \
+                    light cycle sequence from the movie Tron. You can \
+                    play against AI opponents, against other human \
+                    users over the network, or a mixture of both.
+
+subport ${name}-common {
+    description     common files for the $pretty_name packages
+    long_description-append \
+                    \n\nThis package contains files that are common to \
+                    both the ${name} and ${name}-dedicated packages.
+}
+subport ${name}-dedicated {
+    description     dedicated game server for $pretty_name
+    long_description-append \
+                    \n\nThis package installs the dedicated network \
+                    server for the $pretty_name game. It allows you to \
+                    host games that other online players can connect \
+                    to, and play multi-player matches. You can also \
+                    host online multi-player matches directly from \
+                    inside the game, but the dedicated server allows \
+                    you to continue hosting matches while not actively \
+                    running the game.
+}
+
+distfiles           ${distname}${extract.suffix}:${name}
+checksums           ${distname}${extract.suffix} \
+                    rmd160  76eed9dd9fe6d888eba0fae67696040ec1eccb7d \
+                    sha256  59b6c7c01ce3f8cca5437e33f974a637529541a11aa4f52c1a5c17499e26f6a1 \
+                    size    1943758
+
+if {$subport ne "${name}-common"} {
+    master_sites-append http://deb.debian.org/debian/pool/main/a/${name}:debian_pkg
+    distfiles-append    ${name}_${version}-2.debian.tar.xz:debian_pkg
+    checksums-append    ${name}_${version}-2.debian.tar.xz \
+                        rmd160  cc1151617b372d61b6b02b647adb699fdfba3ea0 \
+                        sha256  28b778440ee2d7a8657fec2a8a468723017645a98259a7afac9fa67841a6c36f \
+                        size    29852
+}
+
+depends_build-append    port:autoconf \
+                        port:automake \
+                        port:pkgconfig
+
+if {$subport eq "${name}-common"} {
+    set depends_type    build
+    boost.depends_type  $depends_type
+} else {
+    set depends_type    lib
+}
+depends_$depends_type-append \
+                        port:libxml2
+if {$subport ne "${name}-dedicated"} {
+    depends_$depends_type-append \
+                        port:freetype \
+                        port:libsdl \
+                        port:libsdl_image \
+                        port:libsdl_mixer \
+                        port:libpng
+}
+if {$subport ne "${name}-common"} {
+    depends_run-append  port:${name}-common
+}
+
+post-patch {
+    if {$subport ne "${name}-dedicated"} {
+        reinplace "/LIBS -framework GLU/a\\
+            \\        LIBS=\"\$LIBS -framework OpenGL\"\\
+        " \
```

Comment:
As you know from feedback in previous PRs, patchfiles should be used instead of reinplaces unless the reinplace makes the same replacement a prohibitively large number of times or a variable substitution is needed.

Reply:
Indeed, and my other PRs have been using more patchfiles, for example in #13516. I was hoping that since this reinplace is only adding a single line, it would be ok? :sweat_smile:

Reply:
Only adding a single line is an ideal time to use a patchfile!